frontend-react/src/content/live.json

```diff
@@ -1,70 +1,4 @@
 {
-  "general": {
```

Comment:
I removed data that was needed for the CdcMap component, but left the states data since the React page uses that to build an html list.